sportsref/ncaaf/teams.py

```diff
@@ -75,4 +75,28 @@ def sos(self, year):
             print "ERROR: NO SRS FOUND FOR {} in {}".format(self.teamID, year)
             return np.nan
 
-# TODO - BEGIN HERE
+    def get_roster(self, year):
+        """ Returns the roster for a team for the year
+
+        :param year: the year of the roster
+        :return: a dataframe with the player_id, class, and position
+        """
+        url = sportsref.ncaaf.BASE_URL +\
+              '/schools/{}/{}-roster.html'.format(self.teamID, str(year))
+        doc = pq(sportsref.utils.getHTML(url))
+        table = doc('#all_roster')
+        df = sportsref.utils.parseTable(table)
+        return df
+
+def get_all_college_teams():
```

Comment:
the way that the cfb is organized, this page contains all teams that have ever played and the year they started and also the year they stopped. I guess there's multiple ways to skin this cat. I'm also ok trying to do it with the year. I think I may have added such an approach in the nfl sportsref. Which way should i code this up?